ads/24smi.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript, validateData, validateSrcPrefix} from '../3p/3p';
+
+const jsnPrefix = 'https://jsn.24smi.net/';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function _24smi(global, data) {
```

Comment:
please name the function `24smi` instead. It's much easier for use to maintain code by using same name.

Reply:
I think it would be syntax error if I'll remove underscore, because identifiers cannot start by digit.

Reply:
Ahhh, you're right! I guess we can live with the non-match here : )